test/view_component/view_component_test.rb

```diff
@@ -290,29 +288,24 @@ def test_template_changes_are_not_reflected_in_production
     end
 
     render_inline(MyComponent.new)
-
-    ActionView::Base.cache_template_loading = old_value
   end
 
-  def test_template_changes_are_reflected_outside_production
-    old_value = ActionView::Base.cache_template_loading
-    ActionView::Base.cache_template_loading = false
-
+  def test_template_changes_are_reflected_after_invalidate
```

Comment:
What if we moved this test `integration_test.rb` and reloaded the page instead? Then we wouldn't have to call `ViewComponent::Base::CompileCache.invalidate!`, and our test would more closely resemble the behavior we're trying to implement.